services/core-web/src/components/Forms/noticeOfWork/NOWReviewForm.js

```diff
@@ -139,8 +252,14 @@ export class NOWReviewForm extends Component {
               Cancel
             </Button>
           </Popconfirm>
-          <Button className="full-mobile" type="primary" htmlType="submit">
-            {this.props.title}
+          <Button
+            className="full-mobile"
+            type="primary"
+            htmlType="submit"
+            loading={this.props.submitting}
+            disabled={this.props.submitting}
```

Comment:
This line can be removed since the "loading" prop disables the button as well.

Reply:
removed